x/tokenomics/types/types.go

```diff
@@ -1,3 +1,19 @@
 package types
 
 // This file is in place to declare the package for dynamically generated protobufs
+
+// PendingClaimsResult encapsulates the result of settling pending claims. It is
+// intended to be used to represent settled and expired results as unique instances.
+type PendingClaimsResult struct {
+	NumClaims           uint64
```

Comment:
Thoughts on adding an iota on the result type that contained "Settled" or "Expired"?

Reply:
I'm not sure I follow. Can you elaborate?